api/src/pcapi/connectors/serialization/ems_serializers.py

```diff
@@ -33,3 +33,37 @@ class Site(BaseModel):
 class CinemasProgramsResponse(BaseModel):
     sites: list[Site]
     version: int
+
+
+class Ticket(BaseModel):
+    num_caisse: str
+    code_barre: str
+    num_trans: int
+    num_ope: int
+    code_tarif: str
+    num_serie: int
+    montant: float
+    num_place: str
+
+
+class ReservationPassCultureRequestSerializer(BaseModel):
+    num_cine: str  # VenueProvider.venueIdAtOfferProvider
+    id_seance: str  # show_id
+    qte_place: int
+    pass_culture_price: float
+    total_price: float
```

Comment:
On utilise le type décimal pour stocker les prix des stocks en base de données, vaut mieux utiliser le type décimal ici aussi
```suggestion
    pass_culture_price: decimal.Decimal
    total_price: decimal.Decimal
```